### DISCUSSION TITLE
Tylko cenę "fixing I" w zł/kWh ?

### DISCUSSION THREAD

**Discussion by dariusz737** [PiotrMachowski/Home-Assistant-custom-components-TGE#10] [Ideas]
Ciekawa integracja. Ale może warto rozważyć, opcję zawierającą tylko cenę fixing I, wyrażoną w zł/kWh ?

**Comment by KrzysztofHajdamowicz** [comment]
Domyślnie dostępna jest tylko encja z ceną fixing 1, pozostałe trzeba ręcznie włączyć.

**Comment by PiotrMachowski** [comment]
https://github.com/PiotrMachowski/Home-Assistant-custom-components-TGE/releases/tag/v1.1.1
